src/react/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
this file isnt necessary, since eslint on the base project is configured to extend airbnb

Reply:
Thanks, I removed it.